Makefile

```diff
@@ -12,20 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-TOOLS_DIR := tools
-TOOLS_BIN_DIR := $(abspath $(TOOLS_DIR)/bin)
-GOBIN := $(shell go env GOBIN)
+all: build test fmt
```

Comment:
nit: need .PHONY as well?

This may be a stylistic opinion, but one option is to have all .PHONY definitions in a single line at the top of the file. Not something that i particularly have an opinion for, but as long as we're consistent, but throwing out there if that appeals to you.